[OSDOCS#17012]: CQA bare metal ipi workflow 1 (#117340)
openshift/openshift-docs: installing/installing_bare_metal/ipi/ipi-install-installation-workflow.adoc

:context: ipi-install-installation-workflow

toc::[]
[role="_abstract"]

> Review the different methods for installing {product-title} on bare metal and setup your environment for installation.

Reviewer: I don't see anything on this assembly that talks about different install methods (which makes sense - by the time you're on a page in the bare metal IPI install section, you've already decided on this method). I would suggest focusing on the environment setup only:

```suggestion
Before you can install an {product-title} cluster on bare metal, you must set up your environment for installation.
```

(also a nit here that "set up" is two words as a verb, and "setup" is only used as a noun, per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s))
